src/Arcus.Security.Providers.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -84,7 +84,10 @@ public async Task<Secret> GetSecretAsync(string secretName)
                     return null;
                 }
 
-                return new Secret(secretBundle.Value, secretBundle.SecretIdentifier?.Version);
+                return new Secret(
```

Comment:
The Key Vault `SecretBundle` works with `DateTime` instead of `DateTimeOffset`.

Reply:
Looks like it's in UTC so I would map it to a `DateTimeOffset` for others
https://docs.microsoft.com/en-us/cli/azure/keyvault/secret?view=azure-cli-latest#az-keyvault-secret-set-attributes

Reply:
Ok, done.